e2e/pages/collections/developers/developers.readonly.spec.js

```diff
@@ -49,8 +49,8 @@ describe('the Developers Under Certification Ban collection page', () => {
     describe('using date filter', () => {
       it('should filter listing results', () => {
         page.dateFilter.click();
-        page.fromDate.setValue(['01', 'Sep', 'Tab', '2017']);
-        page.toDate.setValue(['01', 'Oct', 'Tab', '2019']);
+        page.fromDate.setValue('09/01/2017')
```

Comment:
I don't understand this. If I type those characters in my browser, the values do not show up in the date picker. This tool is supposed to simulate a user's experience with the form, right? Why does it need to be this way? When I type the stuff in the previous commit, including the number, letters, and tab, it works

Reply:
When I typed these in certification date filter on search page- it worked.
With tab- it wont work as user types month(mm), cursor automatically shifts to days(dd) so no need for tab.

Reply:
I don't like this as a solution, because it doesn't work for me, but if you say it works then I'll go with it